### PR TITLE
@feature/puppeteer

### PR DESCRIPTION
**Pull request by Eler26**

### CODE REVIEW

File: packages/documentation/web-app/index.mdx [hunk 1]
```diff
@@ -51,3 +51,38 @@ https://enjoycss.com/
...
+
+* Create an up-to-date, automated testing environment. Run your tests directly in the latest version of Chrome using the latest JavaScript and browser features.
+
+* Capture a timeline trace of your site to help diagnose performance issues.
+
+* Test Chrome Extensions.
+
+#### Examples 
+
+Assuming you have a checkout of the Puppeteer repo and have run npm i (or yarn) to install the dependencies, the examples can be run from the root folder like so:
```
Comment (margaritaorfanidi) [inline packages/documentation/web-app/index.mdx]:
```suggestion
To use Puppeteer in your project, run:
    npm i puppeteer
 
Save file as example.js:
```js
const puppeteer = require('puppeteer');

(async () => {
  const browser = await puppeteer.launch();
  const page = await browser.newPage();
  await page.goto('https://example.com');
  await page.screenshot({path: 'example.png'});

  await browser.close();
})();
```
Execute script on the command line:
    node example.js




```

File: packages/documentation/web-app/index.mdx [hunk 2]
```diff
@@ -51,3 +51,52 @@ https://enjoycss.com/
...
+
+(async () => {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+  await page.goto('https://example.com');
+  await page.screenshot({path: 'example.png'});
+
+  await browser.close();
+})();
+
```
Comment (margaritaorfanidi) [inline packages/documentation/web-app/index.mdx]:
```

File: packages/documentation/web-app/index.mdx [hunk 3]
```diff
@@ -51,3 +51,52 @@ https://enjoycss.com/
...
+    NODE_PATH=../ node examples/search.js
```
Comment (margaritaorfanidi) [inline packages/documentation/web-app/index.mdx]:
```suggestion
```
```

File: packages/documentation/web-app/index.mdx [hunk 4]
```diff
@@ -51,3 +51,51 @@ https://enjoycss.com/
...
+const puppeteer = require('puppeteer');
+
+(async () => {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+  await page.goto('https://example.com');
+  await page.screenshot({path: 'example.png'});
+
+  await browser.close();
+})();
```
Comment (margaritaorfanidi) [inline packages/documentation/web-app/index.mdx]:
```suggestion
})();
```
```

File: packages/documentation/web-app/index.mdx [hunk 5]
```diff
@@ -51,3 +51,51 @@ https://enjoycss.com/
...
+const puppeteer = require('puppeteer');
+
+(async () => {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+  await page.goto('https://example.com');
+  await page.screenshot({path: 'example.png'});
+
+  await browser.close();
+})(); ```
```
Comment (Intrusive) [inline packages/documentation/web-app/index.mdx]:
```suggestion
})(); 
```
```

**Pull request merged**